website/blog/2021-09-30-weekly-report.md

```diff
@@ -0,0 +1,106 @@
+---
+title: "社区周报｜新晋 committer 一位，功能亮点更新进行中"
+keywords:
+- APISIX
+- 社区周报
+- 贡献者
+- Good first issue
+description: “独行者速，众行者远”。Apache APISIX 社区周报希望可以帮助社区小伙伴们更好地掌握 Apache APISIX 社区的每周进展，方便大家参与到 Apache APISIX 社区中来。
+tags: [Events]
+---
+> 从 9.13 到 9.30, 有 32 位开发者为 Apache APISIX 提交了 93 个 commits。感谢这些小伙伴为 Apache APISIX 添砖加瓦，是你们的无私付出，让 Apache APISIX 项目变得更好！
+<!--truncate-->
+
+## 导语
+
+Apache APISIX 从开源第一天就以社区方式成长，迅速成为全世界最活跃的开源 API 网关项目。这些成就，离不开社区小伙伴们的共同奋斗。
+
+“独行者速，众行者远”。Apache APISIX 社区周报希望可以帮助社区小伙伴们更好地掌握 Apache APISIX 社区的每周进展，方便大家参与到 Apache APISIX 社区中来。
+
+我们还整理了一些适合新来社区的小伙伴们参加的 issue ！感兴趣的同学们，走过路过不要错过！
+
+## 贡献者统计
+
+![本周贡献者名单](https://static.apiseven.com/202108/1632907894918-c455f40e-a175-4944-8fac-11c590d43786.jpg)
+
+![本周新晋贡献者](https://static.apiseven.com/202108/1632908362102-b0b665e2-f37f-4a82-b8a3-68914925b565.jpg)
+
+## Good first issue
+
+### Issue [#5080](https://github.com/apache/apisix/issues/5080)
+
+问题描述：之前上游服务使用 IP 认证，实际的客户端 IP 是从 x-forwarded-for 请求头中获取的。现在需要改成网关 HMAC 认证，所以需要通过网关屏蔽上游 IP 认证。尝试通过代理重写插件修改 x-forwarded-for，但是没有生效：
+
+![问题描述截图](https://static.apiseven.com/202108/1632799650125-14edb988-f2ad-434d-8d13-04ff3016eb5a.png)
+
+### Issue [#5108](https://github.com/apache/apisix/issues/5108)
+
+问题描述：如下，当在路由上启用请求验证插件:
+
+```
+curl http://127.0.0.1:9080/apisix/admin/routes/35 -H 'X-API-KEY: xxxxxxxxxxxxxxxxxxx' -X PUT -d '
+{
+"uri": "/products/create",
```

Comment:
Please format this JSON data

Reply:
try json formatter online: https://jsonformatter.curiousconcept.com/#

```
{
    "uri":"/products/create",
    "plugins":{
        "request-validation":{
            "body_schema":{
                "type":"object",
                "required":[
                    "productName",
                    "price"
                ],
                "properties":{
                    "productName":{
                        "type":"string"
                    },
                    "price":{
                        "type":"number"
                    }
                }
            }
        }
    },
    "upstream":{
        "service_name":"PRODUCTSSERVICE",
        "type":"roundrobin",
        "discovery_type":"eureka"
    }
}’
```